Cubical/Categories/Profunctor/General.agda

```diff
@@ -39,13 +37,18 @@ open Category
 open Functor
 open UniversalElement
 open Bifunctor
+open NatTrans
 
 -- A profunctor, also called a distributor is a generalization of a
 -- functor where the values are not objects of the codomain, but
 -- instead presheaves
 Profunctor : (C : Category ℓC ℓC')(D : Category ℓD ℓD') → ∀ ℓS → Type _
 Profunctor C D ℓS = Functor C (PresheafCategory D ℓS)
 
+module _ (C : Category ℓC ℓC') (D : Category ℓD ℓD') ℓS where
+  PROFUNCTOR : Category _ _
+  PROFUNCTOR = FUNCTOR C (PresheafCategory D ℓS)
```

Comment:
Composition in this category doesn't use ends?

Reply:
This is a category whose objects are profunctors and morphisms are natural transformations. Maybe you are thinking of the 2-category whose objects are categories and morphisms are profunctors?